pkg/core/resources/apis/mesh/dataplane_validator.go

```diff
@@ -280,6 +280,9 @@ func validateTransparentProxying(tp *mesh_proto.Dataplane_Networking_Transparent
 			if (backendRef.Name != "" || backendRef.Namespace != "") && len(backendRef.Labels) > 0 {
 				result.AddViolationAt(path.Index(i).Field("labels"), "labels cannot be defined when name is specified")
 			}
+			if len(backendRef.Labels) > 0 && backendRef.Port != nil {
```

Comment:
why can't we use port and labels at the same time?

Reply:
My thought was to do not filter out some services by port but I think we can do it